### DISCUSSION TITLE
Standard self installer

### DISCUSSION THREAD

**Discussion by HostileRespite** [ShishirPatil/gorilla#103] [Ideas]
Can we please get an installer for this program? I'm not a programmer and struggle constantly to get these LLMs to work. I spend far more time cursing at terminal errors than I do testing models.

**Comment by ShishirPatil** [comment]
Thanks for this feedback @HostileRespite ! Are you referring to the `gorilla-cli` or just the gorilla-llms through chat completions? if it is the latter, we have this hosted [colab](https://colab.research.google.com/drive/1DEBPsccVLF_aUnmD0FwPeHFrtdC0QIUP?usp=sharing), where you can run the program without any installs required locally. Let us know if you got a chance to try it. 

Also, if you could share which platform (device/operating system) that would be super-helpful for us to release the appropriate installer!

**Reply by chronuz11** [reply]
Same here. Not a programmer, just a fascinated guy screaming at terminals before i can get them all working. Would be windows for me. but I'll manage somehow. thank you for sharing Gorilla! you guys are amazing

**Reply by adistomar** [reply]
@chronuz11 I'm working on a fix for this and was wondering which part is causing difficulty for users: is using the terminal to install gorilla-cli the only issue? Or are you also having difficulty installing Python?
